frontend/app.js

```diff
@@ -1,11 +1,13 @@
 const state = {
   playerData: {
-    str: 17,
```

Comment:
I prefer the trailing comma rule, that way you don't have separate Git diffs for adding entries